php/class-plugin.php

```diff
@@ -391,8 +391,12 @@ public function print_templates() {
 					<img src="{{ data.featured_image.sizes.thumbnail.url }}">
 					{{{ data.text }}}
 				</span>
-			<# } else { #>
+			<# } else if ( data.disable_edit ) { #>
```

Comment:
In case of search result there are id-s present but the editing should be disabled in the selection list. I'll check the other existing properties and see if there's something to use instead of `disable_edit`.